### PR TITLE
Token Payload Rules Page

### PR DESCRIPTION
**Pull request by emmafoci**
- added a documentation page for the Token Payload Rules under Requests

### CODE REVIEW

**Review by ReubenKogie — APPROVED** [review]

File: _pages/docs/request-match-rules/token-payload-rules.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+![Scenario Overview](../../../assets/images/orbital-ui/scenario-overview-token.png)
+
+#### Adding a Token Payload Rule
+
+![Token Payload Match - Request](../../../assets/images/request-match-rules/adding-token-payload-match-rule.png)
+
+In "Token Payload Match Rule" scenario select a "Request" tab and chose a "Token Payload Match Rules" option from the "Request Rule Type" dropdown menu. The image shows the example of key value pair of `Authorization` and `Bearer` with the rule type of `Equals` for the scenario. This rule will check request Token (JWT) Payload to ensure the match. Click + button to add the rule.
+
+Once a request rule have been added, you will need to also add a response. Select "Response" tab. The response includes
```
Comment (glitch452) [inline _pages/docs/request-match-rules/token-payload-rules.md]:
```suggestion
Once a request rule has been added, you will need to also add a response. Select "Response" tab. The response includes
```

File: _pages/docs/request-match-rules/token-payload-rules.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+### Creating a Token Payload Rule in the designer
+
+When creating a new Mock Definition check the Token Validation checkbox. Once a new Mock Definition is created, you start at the Endpoint Overview page. This displays endpoints and existing scenarios, if there are any.
+
+#### Overview of the Endpoints
+
+![Endpoint Overview](../../../assets/images/orbital-ui/endpoint-overview.png)
+
+Select the endpoint to add a scenario, or update an existing one. The following image shows "Default OK Scenario" and "Invalid-Token Scenario" added by default when Token Validation checkbox is selected. Select "JWT Validation" from the dropdown menu. Let's add an "Token Payload Match Rule" scenario for POST `/pets` endpoint.
```
Comment (glitch452) [inline _pages/docs/request-match-rules/token-payload-rules.md]:
```suggestion
Select the endpoint to add a scenario, or update an existing one. The following image shows "Default OK Scenario" and "Invalid-Token Scenario" added by default when Token Validation checkbox is selected. Select "JWT Validation" from the dropdown menu. Let's add a "Token Payload Match Rule" scenario for POST `/pets` endpoint.
```

File: _pages/docs/request-match-rules/token-payload-rules.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+![Token Payload Match - Request](../../../assets/images/request-match-rules/adding-token-payload-match-rule.png)
+
+In "Token Payload Match Rule" scenario select a "Request" tab and chose a "Token Payload Match Rules" option from the "Request Rule Type" dropdown menu. The image shows the example of key value pair of `Authorization` and `Bearer` with the rule type of `Equals` for the scenario. This rule will check request Token (JWT) Payload to ensure the match. Click + button to add the rule.
+
+Once a request rule have been added, you will need to also add a response. Select "Response" tab. The response includes
+the response type (by default Custom), status code (by default - 200), an optional header, and the JSON body `"body": "Response body for Token Payload Match rule"` of the response. The following image shows the example of the response:
+
+![Token Payload Match - Response](../../../assets/images/request-match-rules/adding-token-payload-match-rule-response.png)
+
+Clicking "Save" adds new response to scenario.
```
Comment (glitch452) [inline _pages/docs/request-match-rules/token-payload-rules.md]:
```suggestion
Clicking "Save" adds the new response to the scenario.
```

**Comment by glitch452** [comment]
For the image: adding-token-payload-match-rule.png I would change the example.  The token itself wouldn't have a field of `Authorization` and value of `Bearer`.  Something like a field of `sub` and value of `1234567890` would be more relevant (sub is short subject and the number would represent a user id)

**Review by glitch452 — CHANGES_REQUESTED** [review]
Looks good, just a couple little things.

**Review by glitch452 — APPROVED** [review]

**Pull request merged**